scripts/dts/dtlib.py

```diff
@@ -1156,7 +1156,7 @@ def sub(match):
 
         return _unescape_re.sub(sub, b)
 
-    def _open(self, filename, mode):
+    def _open(self, filename, mode="r", **kwargs):
         # Opens 'filename' in mode 'mode', returning the 'file' object.
         # Searches the directory of the current file and the include path.
```

Comment:
Maybe this doc-comment could be updated to mention that other `open()` arguments can be passed now too.

Reply:
> Maybe this doc-comment could be updated to mention that other open() arguments can be passed now too.

Yep, updated it to say that's a wrapper around open() accepting the same params as it is.